Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserThumbnailView.cpp

```diff
@@ -86,6 +76,19 @@ namespace AzToolsFramework
                         QMenu menu(this);
                         const AssetBrowserEntry* entry = index.data(AssetBrowserModel::Roles::EntryRole).value<const AssetBrowserEntry*>();
                         AZStd::vector<const AssetBrowserEntry*> entries{ entry };
+                        if (m_thumbnailViewWidget->InSearchResultsMode())
+                        {
+                            auto action = menu.addAction("Show In Folder");
```

Comment:
Wrap in tr since this is user-facing string
```suggestion
                            auto action = menu.addAction(tr("Show In Folder"));
```

Reply:
Fixed!